packages/fields-location-google/src/test-fixtures.js

```diff
@@ -10,13 +10,11 @@ export const name = 'Location';
 export { LocationGoogle as type };
 export const supportsUnique = false;
 export const skipRequiredTest = false;
-export const exampleValue = 'ChIJOza7MD-uEmsRrf4t12uji6Y';
-export const exampleValue2 = 'ChIJ_9gDOjmuEmsRB7WRXm_Y6o8';
+export const exampleValue = () => 'ChIJOza7MD-uEmsRrf4t12uji6Y';
+export const exampleValue2 = () => 'ChIJ_9gDOjmuEmsRB7WRXm_Y6o8';
 export const fieldName = 'venue';
 export const subfieldName = 'googlePlaceID';
-export const fieldConfig = {
-  googleMapsKey: process.env.GOOGLE_API_KEY,
-};
+export const fieldConfig = () => ({ googleMapsKey: process.env.GOOGLE_API_KEY });
```

Comment:
should we have default key value for google map too? @timleslie

Reply:
It's not required to get tests to run, so no need for it.